controller_interface/src/controller_interface.cpp

```diff
@@ -119,6 +124,15 @@ std::shared_ptr<rclcpp_lifecycle::LifecycleNode> ControllerInterface::get_node()
   return node_;
 }
 
+std::shared_ptr<rclcpp_lifecycle::LifecycleNode> ControllerInterface::get_node_const() const
```

Comment:
When we really need this method, can we at least reuse the code from the above 'get_node()' method?

Reply:
We run into the same problem as above where get_node() can't be called in a const member function because it's not a const function.